src/vehicle.cpp

```diff
@@ -5613,7 +5554,7 @@ void vehicle::update_time( const calendar &update_to )
 
     // Weather stuff, only for z-levels >= 0
     // TODO: Have it wash cars from blood?
-    if( funnels.empty() && solar_panels.empty() ) {
+    if( funnels.empty() ) {
```

Comment:
Done. Next PR will move that code within `idle`, which will be renamed `tick` and gain time duration parameters